### PR TITLE
Add for each with break method

### PR DESCRIPTION
**Pull request by victorigualada**

### CODE REVIEW

File: lru.d.ts [hunk 1]
```diff
@@ -69,6 +69,9 @@ export class LRUMap<K,V> {
   // Call `fun` for each entry, starting with the oldest entry.
   forEach(fun :(value :V, key :K, m :LRUMap<K,V>)=>void, thisArg? :any) : void;
 
+  // Call `fun` for each entry, starting with the oldest entry. If `fun` returns a falsy value, exit the loop
+  forEachWithBreak(fun :(value :V, key :K, m :LRUMap<K,V>)=>void) : number;
+
```
Comment (justinegreene) [inline lru.d.ts:74]: shouldn't this be added to the non-typescript version also for consistency?

**Review by ralejandre — APPROVED** [review]
Added CHANGELOG, lgtm

**Comment by ralejandre** [comment]
I think this changes look good, maybe this could be PR to open on the main repo.

**Comment by victorigualada** [comment]
Closing since we don't need it anymore

**Pull request closed without merging**